src/components/subscriptions/data/service.js

```diff
@@ -7,11 +7,12 @@ export function createSampleUser(licenseStatus) {
   return {
     userId: faker.random.uuid(),
     emailAddress: faker.internet.email(),
+    pendingSince: moment(faker.date.past(10)).format('MMM DD, YYYY'),
```

Comment:
I'd recommend not formatting the date here, and rather where it gets read (in the table) so that the mock-API here is just returning the raw timestamp.

In terms of the format, I think it should be something like: "June 25, 2020" (full month, day with no leading zero, full year).

Reply:
The date format used here is the same as you mentioned: "June 25, 2020" (full month, day with no leading zero, full year).

Reply:
When I ran your branch locally, I'm seeing dates as "Oct 10, 2015". The month seems to be abbreviated.